src/workspacer.Bar/ConfigContextExtensions.cs

```diff
@@ -10,5 +12,14 @@ public static void AddBar(this IConfigContext context, BarPluginConfig config =
 
             context.Plugins.RegisterPlugin(new BarPlugin(config));
         }
+        
+        public static void AddActionMenu(
```

Comment:
Any reason for this method to return `void`, instead of the plugin created?

Also, it says `AddActionMenu()`, but initiates a `BarPlugin`-indtance. Should we fix naming somewhere to make this more consistent?